### DISCUSSION TITLE
Greenbelt

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#45] [Planning consideration]
# Greenbelt

**Comment by clairemiller2** [comment]
The fundamental aim of Green Belt policy is to prevent urban sprawl by keeping land permanently open; the essential characteristics of Green Belts are their openness and their permanence.
 
Green Belt serves five purposes:
a) to check the unrestricted sprawl of large built-up areas;
b) to prevent neighbouring towns merging into one another;
c) to assist in safeguarding the countryside from encroachment;
d) to preserve the setting and special character of historic towns; and
e) to assist in urban regeneration, by encouraging the recycling of derelict and other urban land.
 
**The general extent of Green Belts across the country is already established. New Green Belts should only be established in exceptional circumstances,** for example when planning for larger scale development such as new settlements or major urban extensions.
 
Once established, **Green Belt boundaries should only be altered where exceptional circumstances are fully evidenced and justified through the preparation or updating of plans**. Strategic policies should establish the need for any changes to Green Belt boundaries, having regard to their intended permanence in the long term, so they can endure beyond the plan period. Where a need for changes to Green Belt boundaries has been established through strategic policies, detailed amendments to those boundaries may be made through non-strategic policies, including neighbourhood plans.
 
**Exceptional circumstances include instances where an authority cannot meet its identified need for homes, commercial or other development through other means. If that is the case, authorities should review Green Belt boundaries** in accordance with the policies in this Framework and propose alterations to meet these needs in full, unless the review provides clear evidence that doing so would fundamentally undermine the purposes (taken together) of the remaining Green Belt, when considered across the area of the plan.
 
Where it is necessary to release Green Belt land for development, plans should give priority to previously developed land, then consider grey belt which is not previously developed, and then other Green Belt locations.
 
The above demonstrates the technical nature of existing Green Belt, in terms of the purposes that it must meet, and the instances in which boundaries can be considered for alteration, as well as when this can be done.
 
When local planning authorities review their Green Belt boundaries, this is a technical exercise which must be completed following a robust, and policy-compliant method. Typically, a Green Belt Review is undertaken in two stages. The first stage examines the performance of a district’s Green Belt in its entirety. While the second stage is more granular and examines the
performance of discrete and small Green Belt parcels, primarily adjacent to existing urban areas or in locations where new settlements are being proposed as part of emerging growth options. A GBR is not a policy or decision-making document that proposes any release of Green Belt land, this is for a council to determine.
 
**As demonstrated throughout the above, Green Belt is designated by local authorities, and decisions made in relation to altering boundaries are made by the local authority, and therefore, any data on the platform should be sourced from local authorities as the authoritative source.**
 
Currently the data is collected, consolidated and published by the Department. This poses several risks.
 

There is a caveat on the platform, which reads '_Contains some data created by MHCLG. We are working to replace it with data from authoritative sources_'. This is fairly hidden, and it is likely that it would be missed by users of the data, while also not providing sufficient context. Therefore, users may download/use the data, which in some instances is likely to be out-of-date, in terms of newly altered Green Belt boundaries. Users may use this data and make decisions using it without knowing it is not wholly accurate.
- e.g. a user downloaded data from the platform to use to complete a Green Belt Review, and when it was shared to confirm with the LPA, they noticed that the Green Belt boundary was incorrect. This might have been due to the LPA sharing incorrect data with MHCLG, or it could have been that any recent updates to the Green Belt wasn't captured in the data issued to MHCLG.
- Similar occurrences may happen for other plan-making scenarios, such as the data not being correct when using to identify and assess sites against site constraints, including land designations such as Green Belt, both within and outside of the LPA boundary.
- Incorrect Green Belt data might also lead to development management risks, both where developers/site promoters/land owners may not have up-to-date information relating to Green Belt boundaries when promoting a site, leading to suitability and deliverability issues, including viability risks. 

We would be very interested to hear from LPAs / users of this data, to understand your view on this matter.

**Reply by clairemiller2** [reply]
Input helpfully provided by colleague @drewhardy77 :

**Data collation update**

- Data is collected through the DELTA platform and is submitted by LPAs on the 31st March each year
- The resulting data is collated and analysed before full release on data.gov.uk (typically) in October of the same year
- Data is primarily collected for annual statistical reporting
- No data is collected in between the annual updates
- When data is released, it is at least 6 months out of date
- At least 84% of green belt by area is under active review following funding release for this purpose in February. This may be more if areas are being supported through local plan delivery funding rather than the specific green belt review fund
- We would expect significant changes as a result of this review and increased housing targets, and relying on annualised data doesn’t meet user needs

**Data quality notes**

- In the process of adding to the platform two errors have been identified
- Areas of Green Belt belonging to four authorities which are now part of North Yorkshire appear as the authority AND as North Yorkshire, thereby giving spatial duplication
- An area in Somerset (created in 2023) has been assigned the old LPA details rather than the merged authority’s
- Both of these errors are only present in the data on the considerations platform, not on the [data.gov.uk](http://data.gov.uk/) version, which is correct. Therefore it seems likely that the errors were a result of our processing of the data rather than inherent in the raw data.

**Reply by clairemiller2** [reply]
In order to progress our thinking on Green Belts, we came up with the following questions to be answered:

**1. Which LPAs should be publishing greenbelt data? Presumably, only a subset of LPAS have a greenbelt?**

@drewhardy77 very helpfully already knew the answer to this one - "176 authorities have at least 10ha of Green Belt present within their boundaries"

**2. What data do they transfer to MHCLG? How? At what cadence? Is there a public task/legal requirement to exchange this data?**

@drewhardy77 was also able to answer this question - "They are required to submit a data return at the end of March each year, primarily for statistical reporting purposes. This is done through DELTA. The data is not published until October the same year. How the data is supplied and what is done to it internally i.e. standardised is yet to be discovered"

**3. Whats the process for a Green Belt Assessment (GBA) - trigger, high level process, time length, outputs (are these internal or published)**

Green belt boundaries should only be reviewed through plan-making. Many LPAs will complete a GBA to provide evidence for their Local Plan, on how different areas of Green Belt perform against the purposes in the NPPF (2024). The LPA then takes the findings into account alongside other evidence in making decisions about the Local Plan strategy, site allocations / broad locations and ultimately possible alterations to Green Belt boundaries.

- Typically, a GBA is undertaken in two stages.
- The first stage examines the performance of a district’s Green Belt in its entirety
- The second stage is more granular and examines the performance of discrete, smaller Green Belt parcels, primarily adjacent to existing sustainable urban areas or in locations where new settlements are being proposed as part of emerging growth options. 
- During Stage 2, site visits are undertaken, to assess the sub-area against the NPPF purposes, to identify boundary features, and to take pictures to characterise the sub-area. 
- Each sub-area is then assessed against each of the Green Belt purposes, often followed by an overall purpose assessment, a wider impact/harm assessment, a boundary assessment, and then recommendations
- Following the updates to the NPPF, a grey belt assessment might also be completed, identifying the sub-areas which may be grey belt

A GBA doesn't automatically lead to the release of Green Belt land, but instead informs decisions made during the plan-making process, and provides robust evidence to support examination. The GBA will be made available on the LPA website, and the assessment process will be included within this. e.g. [St Albans City & District Council GBA](https://www.stalbans.gov.uk/green-belt-documents)

The full GBA takes 6-12 months to complete.

Outside of plan-making, when planning applications come forward for development in the Green Belt, local planning authorities should ensure "that substantial weight is given to any harm to the Green Belt, including harm to its openness. Inappropriate development is, by definition, harmful to the Green Belt and should not be approved except in very special circumstances", but the NPPF does provide instances where development might not be deemed 'inappropriate'. e.g. agricultural buildings, replacement of existing building of same use and scale. @elliotrowen might be able to give more info here.

**Reply by elliotrowen** [reply]
From a Development Management perspective, I believe a Green Belt Assessment (GBA) would typically form part of the decision-making process for an application within the Green Belt, as this would set out why a site wasn't deemed suitable for development previously. However, applications that are submitted for development in the Green Belt must also be accompanied by a ‘very special circumstances’ case, which sets out why the proposals should be considered appropriate development in this context. As I understand it, even developments that meet one of the criteria in paragraph 154 would still require a planning application.

**Reply by drewhardy77** [reply]
> **Data quality notes**
> 
> * In the process of adding to the platform two errors have been identified
> * Areas of Green Belt belonging to four authorities which are now part of North Yorkshire appear as the authority AND as North Yorkshire, thereby giving spatial duplication
> * An area in Somerset (created in 2023) has been assigned the old LPA details rather than the merged authority’s
> * Both of these errors are only present in the data on the considerations platform, not on the [data.gov.uk](http://data.gov.uk/) version, which is correct. Therefore it seems likely that the errors were a result of our processing of the data rather than inherent in the raw data.

These issues have now been corrected

**Comment by drewhardy77** [comment]
Update: having spoken to DELTA, the spatial data is received as part of their annual statistical update. The spatial data is generally supplied to validate statistical returns. There is some internal cleaning e.g. reprojection and self-intersection checking.

If we want to move to a real time update scenario, it would be best to be able to eliminate the need for manual checks, though the data is not complex and doesn't have many attributes, as it's essentially a binary spatial function i.e. is this land green belt - yes/no. The only required addition is the originating authority. One advantage with this data is that every originating authority is already required to send their spatial data at a specified time, so moving to a rolling update at a specified endpoint is not the biggest leap of faith, even if they're not ODP members.
